index.js

```diff
@@ -13,6 +14,7 @@ module.exports = (target, opts) => {
 	let appArgs = [];
 	let args = [];
 	const cpOpts = {};
+	const isWSL = (os.type() === 'Linux' && os.release().indexOf('Microsoft') > -1);
```

Comment:
This check is invalid. `os.release()` doesn't return anything with `Microsoft` for me when in WSL. Did you test this?

Instead use https://github.com/sindresorhus/is-wsl